src/buffer.rs

```diff
@@ -192,16 +192,14 @@ impl<'a, 'b> LayoutRunIter<'a, 'b> {
             .sum();
         let top_cropped_layout_lines =
             total_layout_lines.saturating_sub(buffer.scroll.try_into().unwrap_or_default());
-        let maximum_lines = buffer
-            .height
-            .checked_div(buffer.metrics.line_height)
-            .unwrap_or_default();
+        let maximum_lines = (buffer.height / buffer.metrics.line_height) as i32;
```

Comment:
We may want to still check here?

Reply:
Yep, line height could by accident be 0

Reply:
It should be handled in 22b8a534facb41a7a92156def84c7ad7e2614b84.